deno/rest/v10/interactions.ts

```diff
@@ -131,6 +131,7 @@ export type RESTPatchAPIApplicationGuildCommandResult = Omit<APIApplicationComma
 export type RESTPutAPIApplicationGuildCommandsJSONBody = (
 	| Omit<RESTPostAPIChatInputApplicationCommandsJSONBody, 'dm_permission'>
 	| Omit<RESTPostAPIContextMenuApplicationCommandsJSONBody, 'dm_permission'>
+	| Pick<Partial<APIApplicationCommand>, 'id'>
```

Comment:
this isn't correct, it should append the id as optional to each union element

Reply:
But the types `RESTPostAPIChatInputApplicationCommandsJSONBody` and `RESTPostAPIContextMenuApplicationCommandsJSONBody` are being used in other types, it would affect these 👀

Reply:
It should append the id to each element in the union. I never said it should add it to the post interfaces directly

Reply:
The first one, and yes